File: java-programs/EvenOdd.java
// LBP002
import java.util.Scanner;

public class EvenOdd {
    public static void main(String[] args) {
        System.out.println("Enter an integer");
        Scanner sc = new Scanner(System.in);
        int userInput = sc.nextInt();
        if (userInput >= 0) {
            if (userInput % 2 == 0) {
                System.out.println("even");
            } else {
                System.out.println("odd");
            }
        } else {
            System.out.println("invalid");
        }
    }
}

File: c-programs/EvenOdd.c
// LBP001
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main()
{
  int n;
  // printf("Enter an integer\n");
  scanf("%d", &n);
  if (n >= 0)
  {
    if (n % 2 == 0)
    {
      printf("even");
    }
    else
    {
      printf("odd");
    }
  }
  else
  {
    printf("invalid");
  }
  return 0;
}
